incent-config/Dangerfile

```diff
@@ -99,7 +111,9 @@ def strip_doc(doc)
   doc.strip.tr("\n", ' ')
 end
 
-fail('Unknown program-set. Are the branch and PR named correctly?') if program_set.empty?
+if program_set.empty? || !program_set_from_branch.eql?(program_set_from_commit)
```

Comment:
Great, I love that we're checking the branch and the commit here. Might I suggest that the PR title is included as well? What if the `#program_set` method took all of this into account?

```rb
def program_set_from_pr_title
  return 'na' if not_applicable_pr_title?

  @program_set ||=
    directories.find { |dir| github.pr_title.include?("[#{dir}]") }.to_s
end

def program_set_candidates
  [
    program_set_from_pr_title,
    program_set_from_branch,
    program_set_from_commit
  ].compact.uniq
end

def program_set_mismatch?
  program_set_candidates.length == 1
end

def program_set
  program_set_candidates.first
end
```

Reply:
:+1: Great idea - I'm already checking for that in another place, so I'll add it here, too.